topology/Makefile.am

```diff
@@ -19,6 +19,7 @@ DEPS = \
 MACHINES = \
 	sof-cht-nocodec.tplg \
 	sof-cht-max98090.tplg \
+	sof-apl-hda.tplg \
```

Comment:
sof-apl-hdmi ?
This is not related to HDaudio, if you wanted to deal with an HDaudio codec you'd need additional PCMs and outputs.

Reply:
I'm using sof-apl-hda because I use it to make general DSP HDA driver (skl_hda_generic) work with SOF.  After HDMI works, I'll add analog info into this m4 file.  

Thanks
Mengdong